prep-javascript-functions/main.js

```diff
@@ -0,0 +1,119 @@
+/* Line 53: The log method of the console.object is being called with two arguments, a string &
+ sumTimesFive. */
+const sumTimesFive = addAndMultiplyBy5(12, 6);
+console.log('sumTimesFive exercise:', sumTimesFive); //90
+
+//#5 multiplyAndDivideBy5
+// Line 57: A function named multiplyAndDivideBy5 is being defined with two parameters, num1 & num2.
+function multiplyAndDivideBy5(num1, num2) {
+  return (num1 * num2) / 5;
+}
+/* Line 64: The function multiplyAndDivideBy5 is being called with the arguments 15 & 10, and the value
+ of that call is being assigned to the variable productDivideByFive. */
+/* Line 65: The log method of the console.object is being called with two arguments, a string &
+ productDivideByFive. */
+const productDivideByFive = multiplyAndDivideBy5(15, 10);
+console.log('productDivideByFive exercise:', productDivideByFive); //30
+
+//#6 subtractTwoNumbers
+// Line 69: A function named subtractTwoNumbers is being defined with two parameters, num1 & num2.
+function subtractTwoNumbers(num1, num2) {
+  return num1 - num2;
+}
+/* Line 76: The function subtractTwoNumbers is being called with the arguments 25 & 15, and the value
+ of that call is being assigned to the variable difference. */
+/* Line 77: The log method of the console.object is being called with two arguments, a string &
+ difference. */
+const difference = subtractTwoNumbers(25, 15);
+console.log('subtractByTwoNumbers exercise:', difference); //10
+
+//#7 getCircleCircumference
+// Line 81: A function named getCircleCircumference is being defined with one parameter, radius.
+function getCircleCircumference(radius) {
+  const circumference = 2 * Math.PI * radius;
+  return circumference;
+}
+/* Line 89: The function getCircleCircumference is being called with the argument 10, and the value
+ of that call is being assigned to the variable getCircleCircumferenceResult. */
+/* Line 90: The log method of the console.object is being called with two arguments, a string &
+ getCircleCircumferenceResult. */
+const getCircleCircumferenceResult = getCircleCircumference(10);
+console.log('getCircleCircumference exercise:', getCircleCircumferenceResult); //62.831....
+
+//#8 getFullName
+// Line 95: A function named getFullName is being defined with two parameters, firstName & lastName.
+function getFullName(firstName, lastName) {
+  return firstName + lastName;
+}
+/* Lines 103 & 104: The strings Tré & Mitchell are being assigned to the variables firstName &
+lastName. */
+/* Line 105: The result of the expression firstName concatenated with a string, being concatenated
+ with lastName is assigned to the variable getFullNameResult. */
+/* Line 106: The log method of the console.object is being called with two arguments, a string &
+ getFullNameResult. */
+const firstName = 'Tré';
+const lastName = 'Mitchell';
+const getFullNameResult = firstName + ' ' + lastName;
+console.log('getFullName exercise:', getFullNameResult); //Tré Mitchell
```

Comment:
This should be done using the `getFullName` function. So I should be able to call `getFullName('Shawn', 'Kost')` and it will return `'Shawn Kost'`